pkg/registration/utilities.go

```diff
@@ -0,0 +1,45 @@
+package registration
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	_ "embed"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+)
+
+var (
+	//go:embed scc_public_key.pem
+	publicKey []byte
+)
+
+func decodeBase64(input []byte) ([]byte, error) {
+	decoded := make([]byte, base64.StdEncoding.DecodedLen(len(input)))
+	size, err := base64.StdEncoding.Decode(decoded, input)
+
+	if err != nil {
+		return nil, fmt.Errorf("base64: %s", err)
+	}
+
+	// We need to cut at the end since it is possible to include
+	// gibberish after len(input)
+	return decoded[:size], nil
+}
+
+func sCCPublicKey() (*rsa.PublicKey, error) {
```

Comment:
The naming feels really off 😄  What about `sccPublicKey`?

Reply:
you are right. I don't know what happened. I probably had it public and then made it private :D